script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_1_3.txt

```diff
@@ -17,22 +17,22 @@ According to a rumor, she set a new school record in the physical fitness test t
 "I'm sorry. I'm in the middle of a serious conversation."
 "Huh? A 'serious conversation', is that so?"
 <Page6>
-Unfortunately, I'm neither her trainer nor her rival athlete.
+Unfortunately, I'm neither her trainer nor a rival athlete.
 And you don't just wake up one day and suddenly master the art of running.
 <Page7>
 "Ah, no, but you see, I don't know what you're serious about!"
 "That's true. I don't understand either."
 <Page8>
 How can I put into words the psychology of being attracted to something beautiful?
 However, isn't there something wrong if we can't adequately describe beauty? If we can't honestly admit that something is beautiful?//しかし、言葉にできない美しさというが、美しいものを素直に美しいと言えない状況も、また間違っているのではないだろうか。
-When will human beings be able to transcend all prejudices and discriminations and talk about the essence of things?
+When will human beings be able to transcend all prejudices and discrimination and talk about the essence of things?
 <Page9>
 "Gah!?"
-"T-Tohno-kun, are you all right!?
-You just got hit in the head with something!?"
+"T-Tohno-kun, are you all right?!
+You just got hit on the head with something!"
 "...Yeah, I guess I'm alright."
 <Page10>
-While holding back the pain in the side of my head, I pick up the object that fell on the seat.
+While holding back the pain in the side of my head, I pick up the object that had fallen onto the seat.
```

Comment:
This is an odd phrasing: "While holding back the pain in the side of my head," it is not clear if he is trying to ignore the pain or if he is holding the side of his head because it's sore.
Suggest instead if the former: "While trying to ignore my sore head,"
If the latter: "While nursing my sore head,"

Reply:
Exact verb is こらえつつ
>  (こらえる) (v1,vt) (1) (uk) to bear; to stand; to endure; to put up with; (v1,vt) (2) (uk) to restrain; to control; to keep a check on; (v1,vt) (3) (uk) to forgive; to put up with; to pardon; 
> つつ (prt) (1) while; (prt) (2) even though; despite; (P); 

so while enduring / putting up with, but no physical action of holding